pettingzoo/classic/chess/chess_utils.py

```diff
@@ -324,18 +323,31 @@ def get_observation(orig_board, player: int):
       }
     """
     # from 0-63
-    # Adjust the row number for the white pawn to the 8th if the en passant flag is set, and vice versa for black pawns.
-    square = board.ep_square  # (int) where the en passant happened
+    # Adjust the row number for the white pawn to the 1st if the en passant flag is set, and vice versa for black pawns.
+    # For example
+    # If the white play an en passant move, the opponent can play a special move called en passant capture.
+    # To show this, we denote the pawn at (row, col) = (1, `dest_square`) instead of (5, `dest_square`).
+    square = board.ep_square  # square where the en passant happened (int)
     if square:
-        ours = square < 32
+        ours = (
+            square < 32
+        )  # Less than 32 is a white square, otherwise it's a black square
         row = square % 8
         dest_col_add = 0 if ours else 8 * 7
         dest_square = dest_col_add + row
         if ours:
-            result[base + 0].remove(square + 8)
-            result[base + 0].add(dest_square)
+            result[base + 0].remove(
```

Comment:
Same with the comments in the line being made ugly, just move them to above

Reply:
My typo again. Now I move the comment to above.